database/migrations/000057_encrypted_redirect_jsonb.up.sql

```diff
@@ -0,0 +1,17 @@
+-- Copyright 2024 Stacklok, Inc
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--      http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+
+-- add columns for new encrypted data format
+
+ALTER TABLE session_store ALTER COLUMN encrypted_redirect TYPE JSONB;
```

Comment:
uhm... wait, wouldn't we need to migrate the existing entries to something that that `JSONB` would parse before doing this?

Reply:
As discussed - this column is not used right now.